pkg/agent/plugin/workloadattestor/k8s/k8s.go

```diff
@@ -628,19 +628,26 @@ func lookUpContainerInPod(containerID string, status corev1.PodStatus) (*corev1.
 	return nil, containerNotInPod
 }
 
-func getPodImages(containerStatusArray []corev1.ContainerStatus) map[string]bool {
-	podImages := make(map[string]bool)
-
-	// collect container images
+func getPodImageIdentifiers(containerStatusArray []corev1.ContainerStatus) []string {
+	var podImages []string
```

Comment:
I think this was `map[string]bool` before to avoid duplicate selectors when the same image appears multiple times

Multiple images could have the same selectors, and in the case that an image is scheduled using the sha256, it is even possible that `status.ImageID == status.Image`

Reply:
Ah OK. I wondered why a slower KV collection type was chosen when the V wasn't ever used.

I don't think dupes would actually cause any issues but I'll add it back.

Reply:
@caleblloyd thx for calling that out, fixed.